### PR TITLE
ui: add some vertical space on custom graph page

### PR DESCRIPTION
**Pull request by RaduBerinde**
The custom graph page has an annoying usability issue: if the window
is small enough, or there are enough metrics selected, you cannot
scroll the page up to make room for opening the drop down menu. You
can scroll the page only when the menu is open, and then the page
moves back down when you close it.

This change just adds some `<br>`s to add some blank space in the
page.

Release note: None
Epic: none

### CODE REVIEW

**Comment by dhartunian** [comment]
@koorosh can you take a look and see if there's some CSS we can add that achieves what Radu wants here? The menu should not pop up into a place where you can't scroll. Maybe some of the antd changes you're about to merge fix this already.

**Comment by koorosh** [comment]
It relies on <Select> component from "react-select" package directly which in addition very outdated version in Db Console.
I'd suggest to switch to `Antd` version of `Select` component which opens popup menu to the top so it's never rendered outside of viewport. It might break existing styles which but should be relatively easy to replace.

`antd`'s <Select> component already used in `<ActivateStatementDiagnosticsModal>` component but I'd wait until the merge of `antd` version upgrade and then reuse this component (as this particular component got some major changes).

**Comment by RaduBerinde** [comment]
Ok to merge this and backport in the meantime? This page is used heavily during various escalations.

**Comment by RaduBerinde** [comment]
CC @sean-

**Comment by sean-** [comment]
This has been a usability scourge since time eternal.  Perfect is the enemy of good here and anything is an improvement over the current state of affairs.

**Review by dhartunian — APPROVED** [review]

**Comment by RaduBerinde** [comment]
TFTR!

bors r+

**Pull request merged**
